scripts/workers/submission_worker.py

```diff
@@ -264,7 +266,11 @@ def load_challenge(challenge):
     # make sure that the challenge base directory exists
     create_dir_as_python_package(CHALLENGE_DATA_BASE_DIR)
     phases = challenge.challengephase_set.all()
-    extract_challenge_data(challenge, phases)
+    challenge_module = extract_challenge_data(challenge, phases)
+    if isinstance(challenge_module, ModuleType):
+        return
+    else:
+        sys.exit(1)
```

Comment:
Why exiting the process? So basically what will happen here is that if the module is not loaded then the process will exit without showing any error. This doesn't sound right to me.